class1.html

```diff
@@ -50,9 +50,17 @@
           <p class="green" style="font-size: 200%"> Slide Deck </p>
           <p><a href="http://catherinerust.com/intro/class1.html#/" target="_blank" alt="Class 1 Slides">Class 1 Slides </a></p>
           <p>catherinerust.com/intro/class1.html</p>
+          <p><a href="http://catherinerust.com/intro/links.html" target="_blank">catherinerust.com/intro/links.html</a></p>
```

Comment:
This actually sends me to: `http://catheraaine.com/intro/links.html` - Should it just be that link instead then?